app-text/mupdf/patches/mupdf-1.19.1.patchset

```diff
@@ -44,19 +35,7 @@ index 0fdaecb..926291a 100644
 +	SYS_ZLIB_CFLAGS := $(shell pkg-config --cflags zlib)
 +	SYS_ZLIB_LIBS := $(shell pkg-config --libs zlib)
 +  endif
-+
-+  HAVE_SYS_LEPTONICA := $(shell pkg-config --exists 'lept >= 1.7.4' && echo yes)
-+  ifeq ($(HAVE_SYS_LEPTONICA),yes)
-+	SYS_LEPTONICA_CFLAGS := $(shell pkg-config --cflags lept)
-+	SYS_LEPTONICA_LIBS := $(shell pkg-config --libs lept)
-+  endif
-+
-+  HAVE_SYS_TESSERACT := $(shell pkg-config --exists 'tesseract >= 4.0.0' && echo yes)
-+  ifeq ($(HAVE_SYS_TESSERACT),yes)
-+	SYS_TESSERACT_CFLAGS := $(shell pkg-config --cflags tesseract)
-+	SYS_TESSERACT_LIBS := $(shell pkg-config --libs tesseract)
-+  endif
-+
```

Comment:
I think they could be preserved, works for me here to have these 2 included :)

Reply:
I will test with them deleted from the thirdparty directory, pretty sure they aren't needed though

Reply:
Will do a check here too, for the meantime I looked at a few other recipes around used by other OS's, they keep leptonica and tesseract also in thirdparty:
1. https://packages.altlinux.org/en/p10/srpms/mupdf/specfiles/#line-54
2. https://github.com/archlinux/svntogit-community/blob/packages/mupdf/trunk/PKGBUILD#L32

Reply:
During build with leptonica and tesseract thirdparty included, libraries not included in the recipe, I don't see any use of them during build, so I guess it's safe to move forward without them :)